CHANGELOG.md

```diff
@@ -1,4 +1,30 @@
+* compute: added `next_hop_origin `,`next_hop_med ` and `next_hop_inter_region_cost ` output fields to `google_compute_route` resource ([#8321](https://github.com/hashicorp/terraform-provider-google-beta/pull/8321))
+* compute: added enum STATEFUL_COOKIE_AFFINITY and `strong_session_affinity_cookie` field to `google_compute_backend_service` and `google_compute_region_backend_service` resource ([#8296](https://github.com/hashicorp/terraform-provider-google-beta/pull/8296))
+* compute: added `TDX` instance option for `confidential_instance_type` in `google_compute_instance` ([#8320](https://github.com/hashicorp/terraform-provider-google-beta/pull/8320))
+* compute: promoted `client_destination_port` and `instance` fields in `google_compute_region_network_endpoint` resource to GA (revert) ([#8305](https://github.com/hashicorp/terraform-provider-google-beta/pull/8305))
+* containeraws: added `kubelet_config` field group to the `google_container_aws_node_pool` resource ([#8326](https://github.com/hashicorp/terraform-provider-google-beta/pull/8326))
+* dataproc: switched to the v1 API for `google_dataproc_autoscaling_policy` resource (beta) ([#8306](https://github.com/hashicorp/terraform-provider-google-beta/pull/8306))
+* sourcerepo: added `create_ignore_already_exists` field to `google_sourcerepo_repository` resource ([#8329](https://github.com/hashicorp/terraform-provider-google-beta/pull/8329))
+* sql: added in-place update support for `settings.time_zone` in `google_sql_database_instance` resource ([#8293](https://github.com/hashicorp/terraform-provider-google-beta/pull/8293))
+* tags: increased maximum accepted input length for the `short_name` field in `google_tags_tag_key` and `google_tags_tag_value` resources ([#8324](https://github.com/hashicorp/terraform-provider-google-beta/pull/8324))
+
+BUG FIXES:
+* bigquery: fixed `google_bigquery_dataset_iam_member` to be able to delete itself and overwrite the existing iam members for bigquery dataset keeping the authorized datasets as they are. ([#8304](https://github.com/hashicorp/terraform-provider-google-beta/pull/8304))
+* bigquery: fixed an error which could occur with service account field values containing non-lower-case characters in `google_bigquery_dataset_access` ([#8319](https://github.com/hashicorp/terraform-provider-google-beta/pull/8319))
+* compute: fixed an issue where the `boot_disk.initialize_params.resource_policies` field in `google_compute_instance` forced a resource recreation when used in combination with `google_compute_disk_resource_policy_attachment` ([#8309](https://github.com/hashicorp/terraform-provider-google-beta/pull/8309))
+* compute: fixed the issue that `labels` is not set when creating the resource `google_compute_interconnect` ([#8284](https://github.com/hashicorp/terraform-provider-google-beta/pull/8284))
```

Comment:
```suggestion
* compute: fixed the issue that `labels` was not set when creating the resource `google_compute_interconnect` ([#8284](https://github.com/hashicorp/terraform-provider-google-beta/pull/8284))
```

Reply:
Technically this PR also added in-place update support for labels, which is not currently called out in these notes - could be added to enhancements?

Reply:
I'd prefer keeping this inline with the GA notes for now.